delivery_gls_asm/models/delivery_carrier.py

```diff
@@ -358,8 +361,11 @@ def gls_asm_send_pickup(self, pickings):
             # We post an extra message in the chatter with the barcode and the
             # label because there's clean way to override the one sent by core.
             body = _(
-                "GLS Pickup extra info:<br/> Tracking number: %s<br/> Bultos: %s"
-            ) % (response.get("_codigo"), vals["bultos"])
+                "GLS Pickup extra info:<br/> "
+                "Tracking number: %(codigo)s<br/> Bultos: %(bultos)s",
+                codigo=response.get("_codigo"),
```

Comment:
Are you sure this is correct?

Reply:
He visto que se usaba así en otros módulos; de todas formas, lo he cambiado para dejarlo como seguro es correcto.